src/fparser/two/tests/test_module_use.py

```diff
@@ -0,0 +1,161 @@
+# -----------------------------------------------------------------------------
```

Comment:
Yes, I was thinking of restructuring so that we would do this for both the tests and the source itself. (`symbol_table.py` is 750 lines now.) I'd do that in a separate PR though if that's OK?

Reply:
Yes, agreed.